src/napari_imagej/utilities/_module_utils.py

```diff
@@ -623,3 +519,79 @@ def info_for(search_result: "jc.SearchResult") -> Optional["jc.ModuleInfo"]:
             info = info.cInfo()
         return info
     return None
+
+
+@JImplements("org.scijava.module.process.PostprocessorPlugin", deferred=True)
```

Comment:
Consider throwing this in its own file.

Reply:
I'll wait on this for now. Maybe if we make another processor, we can make a `processors` file.